src/Illuminate/Foundation/Console/stubs/test.database_migration.stub

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace DummyNamespace;
```

Comment:
@crynobone good point. I copies items from ModelMake but this should not be part of that copy.